sdk/devcenter/azure-developer-devcenter/README.md

```diff
@@ -79,20 +79,25 @@ System.out.println(remoteConnectionResponse.getValue());
 // Tear down the Dev Box when we're finished:
 SyncPoller<BinaryData, Void> devBoxDeleteResponse =
                 devBoxClient.beginDeleteDevBox("myProject", "me", "MyDevBox", null);
-devBoxDeleteResponse.waitForCompletion();        
+devBoxDeleteResponse.waitForCompletion();
 ```
 
 ### Environments Scenarios
 ```java com.azure.developer.devcenter.readme.environments
-EnvironmentsClient environmentsClient =
-                new EnvironmentsClientBuilder()
-                        .endpoint(endpoint)
-                        .credential(new DefaultAzureCredentialBuilder().build())
-                        .buildClient();
+DeploymentEnvironmentsClient environmentsClient =
+                    new DeploymentEnvironmentsClientBuilder()
+                            .endpoint("ENDPOINT")
```

Comment:
Looks like `endpoint` got swapped out for `"ENDPOINT"` here, was this purposeful?

Reply:
yeah, that is wrong. It was auto-generated, weird that it was only for deployment client